### PR TITLE
Multiple message attachments part 4

### PR DESCRIPTION
**Pull request by rowanseymour**
screen shot 2017-05-19 at 16 08 24

screen shot 2017-05-19 at 16 08 08

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -81,8 +81,6 @@ var/
 *.manifest
 *.spec
 
-tags
```
Comment (rowanseymour) [inline .gitignore]: Not sure why this was here and it was causing template/msgs/tags to not be committed

File: temba/contacts/templatetags/contacts.py [hunk 2]
```diff
@@ -191,10 +135,8 @@ def history_class(item):
     obj = item['obj']
     classes = []
 
-    if item['type'] == 'msg':
+    if item['type'] in ('msg', 'broadcast'):
```
Comment (rowanseymour) [inline temba/contacts/templatetags/contacts.py]: So that purged broadcasts are rendered like messages

File: temba/msgs/templatetags/sms.py [hunk 3]
```diff
@@ -6,21 +6,22 @@
...
+    'audio/ogg',
+    'audio/mp3',
+    'audio/m4a',
+    'video/mp4'
+}
+
 
 @register.filter
 def as_icon(contact_event):
-
```
Comment (rowanseymour) [inline temba/msgs/templatetags/sms.py]: We no longer display attachment related information via inbox icons, i.e.

screen shot 2017-05-19 at 15 55 14

File: temba/settings_travis.py [hunk 4]
```diff
@@ -6,4 +6,8 @@
 COMPRESS_ENABLED = True
 COMPRESS_OFFLINE = True
 COMPRESS_CSS_HASHING_METHOD = 'content'
-COMPRESS_OFFLINE_CONTEXT = dict(STATIC_URL=STATIC_URL, base_template='frame.html', debug=False, testing=False)
+COMPRESS_OFFLINE_CONTEXT = dict(STATIC_URL=STATIC_URL,
+                                base_template='frame.html',
+                                brand=BRANDING[DEFAULT_BRAND],
```
Comment (rowanseymour) [inline temba/settings_travis.py]: Lack of this has been causing an error during the compress stage on travis

File: templates/contacts/contact_read.haml [hunk 5]
```diff
@@ -206,8 +206,6 @@
 -block extra-less
   -compress css
 
-    %link{type:'text/css', rel:'stylesheet', href:'{{STATIC_URL}}bower/video.js/dist/video-js.css', media:'all'}
```
Comment (rowanseymour) [inline templates/contacts/contact_read.haml]: This is already included in frame.haml and re-including breaks videojs on IE 11 and below

File: templates/msgs/message_box.haml [hunk 6]
```diff
@@ -179,25 +179,21 @@
...
                         {% get_value object 'text' %}
+
                         - if 'label' in actions
                           .value-labels
                             - for label in object.labels.all
                               %span.label.label-info.lbl{ data-id: '{{label.id}}' }
                                 %a{href:"{% url 'msgs.msg_filter' label.pk%}"}
                                   {{label.name}}
 
-                          - if object.get_flow_id
```
Comment (rowanseymour) [inline templates/msgs/message_box.haml]: This is broken as `Msg. get_flow_id` doesn't exist - but we've agreed we don't want to include this for now as it makes getting rid of flow steps harder

File: temba/contacts/models.py [hunk 7]
```diff
@@ -533,8 +533,11 @@ def get_activity(self, after, before):
         broadcasts = []
         for recipient in recipients:
             broadcast = recipient.broadcast
+            media = broadcast.get_translated_media(contact=self, org=self.org) if broadcast.media else None
+
             broadcast.translated_text = broadcast.get_translated_text(contact=self, org=self.org)
             broadcast.purged_status = recipient.purged_status
+            broadcast.attachments = [media] if media else []
```
Comment (norkans7) [inline temba/contacts/models.py]: Broadcast does not have attachments field

Comment (rowanseymour) [inline temba/contacts/models.py]: yeah this is a dynamic attribute so that purged broadcasts can be more easily rendered like msgs

Comment (norkans7) [inline temba/contacts/models.py]: sawa

**Pull request merged**